firmware/src/core/wifiWidget.cpp

```diff
@@ -55,16 +60,16 @@ void WifiWidget::setup() {
       m_configPortalRunning = true;
       Serial.println("Configuration portal running.");
       m_manager.selectScreen(statusScreenIndex);
-      display.fillScreen(TFT_BLACK);
-      display.drawCentreString("Configure", ScreenCenterX, ScreenCenterY - lineHeight, 4);
+      m_manager.clearScreen();
+      m_manager.drawCentreString("Configure", ScreenCenterX, ScreenCenterY - lineHeight, fontSize);
       m_manager.selectScreen(statusScreenIndex + 1);
-      display.drawCentreString("Connect", ScreenCenterX, ScreenCenterY - lineHeight * 2, 4);
-      display.drawCentreString("phone or PC", ScreenCenterX, ScreenCenterY - lineHeight, 4);
-      display.drawCentreString("to WiFi network:", ScreenCenterX, ScreenCenterY, 4);
-      display.setTextColor(TFT_SKYBLUE);
-      display.drawCentreString(m_apssid, ScreenCenterX, ScreenCenterY + lineHeight, 4);
-      display.setTextColor(TFT_GREENYELLOW);
-      display.drawCentreString("192.168.4.1", ScreenCenterX, ScreenCenterY + lineHeight * 2, 4);
+      m_manager.drawCentreString("Connect", ScreenCenterX, ScreenCenterY - lineHeight * 2, fontSize);
+      m_manager.drawCentreString("phone or PC", ScreenCenterX, ScreenCenterY - lineHeight, fontSize);
+      m_manager.drawCentreString("to WiFi network:", ScreenCenterX, ScreenCenterY, fontSize);
+      m_manager.setFontColor(TFT_SKYBLUE);
+      m_manager.drawCentreString(m_apssid.c_str(), ScreenCenterX, ScreenCenterY + lineHeight, fontSize);
```

Comment:
Why is this `c_str()` required now?
That seems inconvenient.

Reply:
OpenFontRender uses c-Strings.
But I can change the signature to String if that's preferred (or add it as alternative).

Reply:
I've changed the functions to use Strings